src/Init/Data/Char/UnicodeSkipList.lean

```diff
@@ -0,0 +1,100 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jean-Baptiste Tristan
+-/
+prelude
+import Init.Data.Range
+
+open Std
+
+namespace Char.UnicodeSkipList
+
+/-
```

Comment:
Are you aware of the various types of comments we have in Lean? We have `/- ... -/` for normal comments, `/-! ... -/` for module documentation that will be rendered in the online documentation, and `/-- ... -/` for declaration docstrings which appear in the online documentation and also when hovering over the declaration in VS code. For an example, you can look at https://github.com/leanprover/lean4/blob/master/src/Init/Prelude.lean and the corresponding online documentation https://leanprover-community.github.io/mathlib4_docs/Init/Prelude.html. Most well-documented files have a module doc in the beginning, right after the imports and before everything else (so before the `open Std` in this case) and then potentially additional module docs for dividing large files into sections and explaining details relevant to that section. The comment you have here should almost certainly be a module doc, and you could consider turning the comments on the functions below into docstrings.

Reply:
ACK for module doc. I'll add a docstring to search. I did not make the comment for searchRuns and searchOffsets docstrings because ideally, I wouldn't even want those to appear at all in the doc, they're not meant to be called directly by an end user.